modules/ROOT/content-nav.adoc

```diff
@@ -27,6 +27,7 @@
 ** xref:docker/backup-restore.adoc[]
 ** xref:docker/security.adoc[]
 ** xref:docker/ref-settings.adoc[]
+** xref:docker/docker-compose-settings.adoc[]
```

Comment:
We have a tutorial on how to set up a cluster using docker-compose. I wonder if this page should also go there under Tutorials and examples.